reana_server/config.py

```diff
@@ -180,8 +175,12 @@ def _(x):
 
 # Workflow scheduler
 # ==================
-REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY = 60
+REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY = os.getenv(
+    "REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY", 600
+)
```

Comment:
I wonder whether we should move the default production values to Helm as well? It's a bit strange to rely on non-existing env in production?

Reply:
I thought the same, but then the tests were failing as the value was `None`. I can switch it back and mock the tests.

Reply:
I've changed it and the [reana](https://github.com/reanahub/reana/compare/2876fc9dfcc88c886e45e4e568b456c5d67297a0..2235424045d01483343e14cd90ea9e6cc292c37e) one accordingly.